presto-prometheus/src/main/java/io/prestosql/plugin/prometheus/PrometheusMetricHeader.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.prometheus;
+
+import java.util.HashMap;
+
+public class PrometheusMetricHeader
+        extends HashMap<String, String>
```

Comment:
Please don't extend hash map. Create dedicated interface or use hash map directly.

Reply:
Done in 2cea942b (will push with larger group of commits later)